csp-billing-adapter.spec

```diff
@@ -15,28 +15,35 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
```

Comment:
Shouldn't this be provided by the standard rpm macros used by the build?

Reply:
I don't really know the state of python multispec. This is the line I see in the packages I've looked at for reference. And it seems to work as expected :shrug: .

Reply:
Leaving as is for now.

Reply:
Yeah that is fine... And the line has no effect if the macro is already defined...